docs/manual/install/arch.rst

```diff
@@ -11,7 +11,7 @@ Using an AUR Helper
 ===================
 
 The preferred way to install Qtile is with an `AUR helper`_. For example,
-if you use `yaourt`_:
+if you use yaourt:
```

Comment:
Instead of removing the link, it could be updated to https://archlinux.fr/yaourt-en